src/components/base-address/factories/countries-provinces-factory.ts

```diff
@@ -0,0 +1,78 @@
+import countriesData from 'country-list/data.json'
+import provincesData from 'provinces/provinces.json'
```

Comment:
You need a "base-address/package.json" (see other components for examples; set version to 0.0.0 initially) to specify all this component's dependencies.

Check the other files in here for additional dependencies.

Reply:
This is the current base-address/package.json:
![image](https://github.com/bcgov/bcrs-shared-components/assets/144158015/593c4e2d-1170-45d0-8132-6b4bbd3a139e)
Should I reset the version to 0.0.0 and add the relevant dependencies?

Reply:
Oh right, you're updating an existing component.

Leave the version alone and I'll bump it to 3.0.0 when I publish this.